packages/frontend-v2/components/ScheduleCourse/ScheduleCourse.tsx

```diff
@@ -69,52 +68,116 @@ export const ScheduleCourse = forwardRef<
+          transform: hovered ? "scale(1.04)" : "scale(1)",
+        }}
+        onMouseEnter={() => {
+          setHovered(true);
+        }}
+        onMouseLeave={() => {
+          setHovered(false);
+        }}
         ref={ref}
         {...attributes}
-        cursor={isDisabled ? "default" : isDragging ? "grabbing" : "grab"}
-        borderRadius="md"
-        boxShadow="md"
-        backgroundColor={isDisabled ? "neutral.main" : "white"}
-        color={isDisabled ? "primary.blue.light.main" : "gray.800"}
-        pl="2xs"
-        pr="sm"
-        py="xs"
-        width="100%"
-        _hover={{ bg: isDisabled ? "neutral.main" : "neutral.700" }}
-        _active={{ bg: isDisabled ? "neutral.main" : "neutral.900" }}
       >
-        <Flex alignItems="center" {...listeners}>
-          <DragHandleIcon
-            mr="md"
-            color={
-              isDisabled ? "primary.blue.light.main" : "primary.blue.dark.main"
-            }
-          />
-          <Text fontSize="sm" pr="2xs">
-            {getCourseDisplayString(scheduleCourse)}
-          </Text>
-          <Text fontSize="xs" pr="md" noOfLines={1}>
-            {scheduleCourse.name}
-          </Text>
-        </Flex>
-        {isEditable && (
-          <CourseTrashButton
-            marginLeft="auto"
+        <div
+          style={{
+            display: "flex",
+            alignItems: "center",
+            flexGrow: 1,
+            background: "",
+            padding: "8px 8px",
+            cursor: isOverlay ? "grabbing" : "grab",
+          }}
+          {...listeners}
+        >
+          <svg
+            viewBox="0 0 10 10"
+            focusable="false"
+            style={{
+              margin: "0px 8px 0px 0px",
+              width: "1em",
+              height: "1em",
+              display: "inline-block",
+              lineHeight: "1em",
+              WebkitFlexShrink: 0,
+              flexShrink: 0,
+              color: "var(--chakra-colors-primary-blue-dark-main)",
+              verticalAlign: "middle",
+            }}
+          >
+            <path
+              fill="currentColor"
+              d="M3,2 C2.44771525,2 2,1.55228475 2,1 C2,0.44771525 2.44771525,0 3,0 C3.55228475,0 4,0.44771525 4,1 C4,1.55228475 3.55228475,2 3,2 Z M3,6 C2.44771525,6 2,5.55228475 2,5 C2,4.44771525 2.44771525,4 3,4 C3.55228475,4 4,4.44771525 4,5 C4,5.55228475 3.55228475,6 3,6 Z M3,10 C2.44771525,10 2,9.55228475 2,9 C2,8.44771525 2.44771525,8 3,8 C3.55228475,8 4,8.44771525 4,9 C4,9.55228475 3.55228475,10 3,10 Z M7,2 C6.44771525,2 6,1.55228475 6,1 C6,0.44771525 6.44771525,0 7,0 C7.55228475,0 8,0.44771525 8,1 C8,1.55228475 7.55228475,2 7,2 Z M7,6 C6.44771525,6 6,5.55228475 6,5 C6,4.44771525 6.44771525,4 7,4 C7.55228475,4 8,4.44771525 8,5 C8,5.55228475 7.55228475,6 7,6 Z M7,10 C6.44771525,10 6,9.55228475 6,9 C6,8.44771525 6.44771525,8 7,8 C7.55228475,8 8,8.44771525 8,9 C8,9.55228475 7.55228475,10 7,10 Z"
+            ></path>
+          </svg>
```

Comment:
could we move this into its own file/component just for readability

Reply:
Good call, will do.